recipes/treepoem/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "treepoem" %}
+{% set version = "3.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0eaf53efe363fa63eb76aa3a5d322ead79359087119bffcab9f61147307ab087
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
I think this recipe can be noarch.

Reply:
Making it noarch results in failure for windows build, as treepoem.exe gets created, hence removed noarch line.